scheduler/src/cook/kubernetes/api.clj

```diff
@@ -184,6 +184,39 @@
            (-> m (get "cpu") to-double)
            0.0)})
 
+(defn get-taint-key
+  "Get the taint variable out of a taint key out of the V1Taint object."
+  [^V1Taint taint]
+  (.getKey taint))
+
+(defn process-taints
```

Comment:
Taking another look at this, it seems like this would be more clear to have this logic inside `get-node-pool` and `node-schedulable?` instead of building this separate map.

Reply:
Then I'd have to reprocess taints twice, with different logic.  Seemed to make more sense to process once and reuse the result, and that way I only have one process-taints logic.